testbench/__init__.py

```diff
@@ -23,4 +23,4 @@
     rest_server,
 )
 
-run = rest_server._run
+run = rest_server._run()
```

Comment:
I am not sure this change is correct.  Before it was defining `run` as a function that folks could call.  Now it is the result of a function, and implicitly starts the function on load.  Is that what we want?  If so, why?  Can you leave a comment?

Reply:
In the beginning I did this change because I was not able to start storage test bench using waitress from command line if 'run' is a function and not a wsgi server. But as now we are starting testbench programmatically via script, I'll revert this code change as it's not needed.